minigalaxy/ui/gametile.py

```diff
@@ -618,7 +623,7 @@ def __state_updatable(self):
             self.wine_icon.set_margin_left(22)
 
     def __state_updating(self):
-        self.button.set_label(_("updating…"))
+        self.button.set_label(_("Updating…"))
```

Comment:
This change makes it necessary to update all translation files. So we'd either have to leave this change out or update those.